.github/workflows/ansible-test.yml

```diff
@@ -78,51 +78,49 @@ jobs:
             ansible.netcommon
             ansible.utils
 
-# If the application you are testing is available as a docker container and you want to test
-# multiple versions see the following for an example:
-# https://github.com/ansible-collections/community.zabbix/tree/master/.github/workflows
+# Please see the Readme why we disabled integration tests at the moment
 
```

Comment:
# Please consult the Readme for information on why we disabled integration tests temporarily.